src/main/java/seedu/medmoriser/logic/parser/FindCommandParser.java

```diff
@@ -27,7 +30,17 @@ public FindCommand parse(String args) throws ParseException {
 
         String[] nameKeywords = trimmedArgs.split("\\s+");
 
-        return new FindCommand(new QuestionContainsKeywordsPredicate(Arrays.asList(nameKeywords)));
+        String findType = nameKeywords[0];
+        switch (findType) {
+        case "/t":
```

Comment:
Maybe can change to t/ to keep the same format as the add command

Reply:
yes i agree with Jon, this way the user will not be confused.